app/views/proposals/show.html.erb

```diff
@@ -0,0 +1 @@
+<%= render partial: "proposal", locals: { proposal: @proposal}  %>
```

Comment:
Hmm, is that partial used in multiple places? Maybe we can just move that file in here.